### DISCUSSION TITLE
support m3u8 radio stream

### DISCUSSION THREAD

**Discussion by Yumega** [navidrome/navidrome#2455] [Ideas]
please support m3u8 stream , only support mp3 now

**Comment by certuna** [comment]
m3u8 is just a playlist file, what kind of stream is it?

**Reply by Yumega** [reply]
https://stream.hndt.com/live/yule/playlist.m3u8

**Reply by certuna** [reply]
This is a HE-AACv2 stream - works in any browser that supports this codec. I've tested it in Safari on macOS, works.
